models/lorenz_96_tracer_advection/shell_scripts/README

```diff
@@ -0,0 +1,54 @@
+# DART software - Copyright UCAR. This open source software is provided
+# by UCAR, "as is", without charge, subject to all terms of use at
+# http://www.image.ucar.edu/DAReS/DART/DART_download
+#
+# DART $Id$
+
+The scripts in this directory are:
+
+run_filter.csh - 
```

Comment:
this is possibly the only script in this directory that should be kept.  the rest could be deleted and this file chopped down to a few lines.

Reply:
just removed shell_scripts for now.